content/100-getting-started/index.mdx

```diff
@@ -195,3 +197,22 @@ Build real-time applications by subscribing to data changes in your database usi
     link="https://github.com/prisma/prisma-examples?tab=readme-ov-file#prisma-pulse"
   />
 </Grid>
+
+## <Inspect color="#16A394" width="27" height="24" /> Prisma Optimize
+
+Make your database queries faster by using the insights and recommendations generated by Prisma Optimize.
+
+<Grid>
+  <LinkCard
+    icon="fa-solid fa-arrow-right"
+    title="Get started"
+    desc="Add Optimize to your app to generate useful insights and recommendations to improve query performance."
```

Comment:
to gather useful insights and receive recommendations

Reply:
```suggestion
    desc="Add Optimize to your app to gather useful insights and receive recommendations to improve query performance."
```